src/demos/email-validation/email-validation-demo.component.html

```diff
@@ -1,17 +1,60 @@
-<div style="max-width: 50%">
-  <label>
-    Email:
-    <input
-      class="sky-form-control"
-      skyEmailValidation
-      [(ngModel)]="emailValidator"
-      #email="ngModel">
-  </label>
-  <div
-    class="sky-error-label"
-    *ngIf="email.errors && email.errors.skyEmail && (email.dirty && email.touched)">
-    <span [hidden]="!email.errors.skyEmail.invalid">
-      Enter a valid email in the joe@abc.com format.
-    </span>
-  </div>
-</div>
+<sky-fluid-grid>
+  <sky-row>
+    <sky-column
+      screenSmall="6"
+    >
+      <h2
+        class="sky-section-heading"
+      >
+        Reactive form input validation
```

Comment:
Anyone have an opinion on whether `input` should be `email` in this header to be more specific to the component in question? Or is it better to be generic and use the same header on the email validation topic and the URL validation topic? ... This verbiage occurs on 4 headers between the 2 components.

Reply:
Leaving the generic header